### PR TITLE
this is the About us page

### PR DESCRIPTION
**Pull request by NolwaziNgcongo**

### CODE REVIEW

File: AboutUs/index.html [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+    <h3>Want to work with us?</h3>
+    <button onclick="window.location.href='mentor.html'">Mentor</button>
+</section>
+
+<footer>
+    <div class="footer-container">
+      <div class="footer-left">
+        <h2>TechGirlsHub</h2>
+        <p>TechGirlsHub is a team of women in tech, showcasing projects, expertise, and services while connecting with clients.</p>
+        <div class="social-icons">
```
Comment (MlonziNolwazi) [inline AboutUs/index.html:94]: please use icons

File: AboutUs/index.html [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+</head>
+<header>
+    <div class="header-container">
+        <div class="logo">
+          <img src="/images/TechGirls.jpg" alt="TechGirlsHub Logo">
+          <h1>TECHGIRLSHUB</h1>
+        </div>
+        <nav>
+          <ul>
+            <li><a href="#">About Us</a></li>
```
Comment (MlonziNolwazi) [inline AboutUs/index.html:17]: please include the url links,  and  Feedback Rating link - url should be feedbackRating.php

File: AboutUs/index.html [hunk 3]
```diff
@@ -0,0 +1,129 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>About Us</title>
```
Comment (MlonziNolwazi) [inline AboutUs/index.html]: Can we have this as "TechGirlsHub - About Us"

File: AboutUs/index.html [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+            <li><a href="#">Team members</a></li>
+            <li><a href="#">Contact Us</a></li>
+          </ul>
+        </nav>
+      </div>
+</header>
+<h1>About Us</h1>
+
+<section class="hero">
+    <div class="hero-image" style="background-image: url('');">
```
Comment (MlonziNolwazi) [inline AboutUs/index.html]: Avoid having unused code.

**Review by MlonziNolwazi — CHANGES_REQUESTED** [review]
Please do the requested changes before merging ur branch

**Review by techgirlshub — APPROVED** [review]

**Pull request merged**